Spanner/src/Instance.php

```diff
@@ -154,6 +168,7 @@ public function __construct(
         $this->info = $info;
 
         $this->setLroProperties($lroConnection, $lroCallables, $this->name);
+        $this->directedReadOptions = $options['directedReadOptions'] ?? [];
```

Comment:
Can you help me understand what's the difference between and how would a user choose among these options?
1. I don't provide `directedReadOptions` in the `$options` => a default `[]` is chosen
2. I provide `$options['directedReadOptions'] = null` => a default `[]` is chosen
3. I provide `$options['directedReadOptions'] = []` => overrides default with same value
4. But what happens if user passes empty config like: 
    `$options['directedReadOptions'] = ['include_replicas' => [], 'exclude_replicas' => []]`

Reply:
1 to 3 of your question lead to the `Grpc.php` class here https://github.com/googleapis/google-cloud-php/pull/6078/files#diff-23685c926c7fbfb00329decf0e9b0780662f12145c6d9114b6670838aa86bf9fR1608. The function `setDirectedReadOptions` set `directedReadOptions` only if the passed args has an `not empty` `directedReadOptions`